test/cli/repo.js

```diff
@@ -16,4 +16,16 @@ describe('repo', () => runOnAndOff((thing) => {
     const out = await ipfs('repo version')
     expect(out).to.eql(`${repoVersion}\n`)
   })
+
+  it('get repo stats', async () => {
+    const stats = await ipfs('repo stat')
+    expect(stats).to.match(/^RepoSize:\s+\d+$/gm)
+    expect(stats).to.match(/^StorageMax:\s+\d+$/gm)
```

Comment:
```suggestion
    expect(stats).to.match(/^StorageMax:\s\d+$/gm)
```

Only need to match one space here, I think.